pandas/tests/frame/test_constructors.py

```diff
@@ -196,8 +196,10 @@ def test_constructor_overflow_int64(self):
         assert df_crawls['uid'].dtype == np.uint64
 
     @pytest.mark.parametrize("values", [np.array([2**64], dtype=object),
-                                        np.array([2**64]), [2**64]])
-    def test_constructor_overflow_uint64(self, values):
+                                        np.array([2**65]), [2**64 + 1],
```

Comment:
`np.array([2**65])` should be the same as the `np.array([2**64], dtype=object)` case ?

Reply:
Should be but just testing out different values and slightly different initialization of large data.

Reply:
I mean

```
In [70]: np.array([2**64], dtype=object)
Out[70]: array([18446744073709551616], dtype=object)

In [71]: np.array([2**64])
Out[71]: array([18446744073709551616], dtype=object)
```

is twice the same you test?

Reply:
No?  They're two different values.  Just trying out two different large numbers.

Reply:
ah sorry, didn't see the difference in the power term